docs/access-ui.md

```diff
@@ -1,29 +1,48 @@
-# Access UIs
+---
+weight: 300
+toc: true
+title: Access Dashboards
+menu:
+    docs:
+        parent: kube
+images: []
+draft: false
+---
 
-Prometheus, Grafana, and Alertmanager dashboards can be accessed quickly using `kubectl port-forward` after running the quickstart via the commands below. Kubernetes 1.10 or later is required.
+This page assumes that you have installed Prometheus Operator using Kube-Prometheus and want to learn how to access Prometheus, Grafana and Alertmanager dashboards.
```

Comment:
Is this part really necessary? We're already in kube-prometheus docs here 🤔

Reply:
I thought just to mention the reminder but yeah doesn't make that much of a difference.